src/jsMain/kotlin/org/cqfn/diktat/demo/frontend/components/EditorForm.kt

```diff
@@ -161,21 +157,37 @@ class EditorForm : RComponent<Props, CodeFormState>() {
                         input(type = InputType.file, name = "diktat-analysis.yml") {
                             attrs.accept = ".yml,.yaml"
                             attrs.onChangeFunction = { event ->
+                                setState {
+                                    isLoading = true
+                                }
                                 val target = event.target as HTMLInputElement
                                 target.files?.asList()?.firstOrNull()?.let { file ->
                                     val reader = FileReader().apply {
                                         onload = { event: Event ->
                                             val text = event.target.asDynamic().result.toString()
                                             setState {
                                                 codeForm = codeForm.copy(diktatConfig = text)
+                                                isLoading = false
                                             }
                                         }
                                     }
                                     reader.readAsText(file)
                                 }
                             }
+                            attrs.onLoadStart = {
+                                setState {
+                                    isLoading = true
+                                }
+                            }
                         }
                     }
+                    div {
+                         +"Loading..."
+                        attrs.hidden = !state.isLoading
+                    }
+                    div {
+                        +(state.codeForm.diktatConfig?.let{ "Config loaded" } ?: "")
```

Comment:
Maybe add `hidden` for this to?? Now you'll see an empty div. Or you could just unify those two, with conditional text

Reply:
Seems like adding hidden is a good idea in order to make our `index.html` look a bit better

Reply:
Ok, will make it hideable